api/tests/core/offers/test_api.py

```diff
@@ -1195,13 +1196,15 @@ def test_delete_stock_cancel_bookings_and_send_emails(self):
         assert booking3.cancellationReason == BookingCancellationReasons.OFFERER
 
         assert len(mails_testing.outbox) == 3
-        print(mails_testing.outbox)
-        assert mails_testing.outbox[0].sent_data["To"] == "beneficiary@example.com"
-        assert mails_testing.outbox[2].sent_data["To"] == "offerer@example.com"
+        assert set(outbox.sent_data["To"] for outbox in mails_testing.outbox) == set(
+            [booking1.email, booking3.email, offerer_email]
+        )
 
-        assert push_testing.requests[-1] == {
+        last_request = push_testing.requests[-1]
```

Comment:
pour éviter toute mauvaise surprise, j'aurai  même fait:
```suggestion
        last_request = deepcopy(push_testing.requests[-1])
```